cupy/sparse/compressed.py

```diff
@@ -65,6 +65,12 @@ def __init__(self, arg1, shape=None, dtype=None, copy=False):
             if len(data) != len(indices):
                 raise ValueError('indices and data should have the same size')
 
+        elif base.isdense(arg1) and arg1.ndim == 2:
+            data, indices, indptr = self._convert_dense(arg1)
```

Comment:
Another csr_matrix is created to initialize a csr_matrix, a little entangled. An option is to make cusparse.py independent of cupy.sparse.spmatrix suites, but it is followed with some amount of code change. Too much?